cost/azure/unattached_volumes/README.md

```diff
@@ -0,0 +1,58 @@
+# Azure Unattached Volumes
+
+## What it does
+
+This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
+
+If the user approves that the volumes should be deleted, the policy will delete the volumes.
+If the volume is not able to be deleted, say, due to it being locked, the volume will be tagged to indicate the error that was received.
+
+If the issue causing the delete failure is removed, the next run of the policy will delete the volume.
+Note: The unattached volumes report will reflect the updated set of unattached volumes on the subsequent run.
+
+Optionally, the user can specify one or more tags that if found on a volume will exclude the volume from the list.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Unattached days* - The number of days a volume has been unattached.
+- *Email addresses* - A list of email addresses to notify.
+- *Exclude Tags.* - A list of tags used to excluded volumes from the incident.
+- *Create Final Snapshot* - Boolean for whether or not to take a final snapshot before deleting.
+- *Workspace ID* - ID of the log analytics workspace which is created in the Azure portal.
```

Comment:
Why is the parameter needed ?  See other polices that use log analytics.  they don't need the workspace id.

Reply:
The reason we are asking workspace id as user input is, in the list disk API response, we are not getting the workspace id to which the disk is attached to, as in the case of Azure inefficient instance utilization, workspace id was returned in the list instances API response. 
For getting the event logs from log analytics, we are not attaching any disk in the workspace so it will not be possible to directly fetch the workspace id. Since we require workspace id to fetch the event logs, we have made it as user input.

Reply:
ok, thank you.  I'm considering other methods to identify the unattached volumes.  using log analytics may not be the best way.

Reply:
Ok sure. Please let me know if any changes need to be done from my end.